src/features/search/components/__tests__/SearchBox.native.test.tsx

```diff
@@ -76,4 +76,38 @@ describe('SearchBox component', () => {
       })
     )
   })
+
+  it('should not execute a search if input is empty', () => {
+    const { getByPlaceholderText } = render(<SearchBox searchInputID={searchInputID} />)
+    const searchInput = getByPlaceholderText('Offre, artiste...')
+
+    fireEvent(searchInput, 'onSubmitEditing', { nativeEvent: { text: '' } })
+
+    expect(navigate).not.toBeCalledWith(
```

Comment:
techniquement, on veut tester `not.toBeCalled()` plutôt que `not.toBeCalledWith(…)`

Reply:
Ok simplifions :)